doc/releases/changelog-0.9.0.md

```diff
@@ -111,45 +94,78 @@
   always take precedence over global pass pipelines.
 
   The available MLIR passes are listed and documented in the
-  [`catalyst.passes` module documentation](https://docs.pennylane.ai/projects/catalyst/en/stable/code/__init__.html#module-catalyst.passes).
+  [passes module documentation](https://docs.pennylane.ai/projects/catalyst/en/stable/code/__init__.html#module-catalyst.passes).
 
-* A peephole merge rotations pass is now available in MLIR. It can be added to `catalyst.passes.pipeline`,
-  or the Python function `catalyst.passes.merge_rotations` can be directly called on a `QNode`.
+* A peephole merge rotations pass, which acts similarly to the Python-based [PennyLane merge rotations transform](https://docs.pennylane.ai/en/stable/code/api/pennylane.transforms.merge_rotations.html),
+  is now available in MLIR and can be applied to QNodes within a qjit-compiled function.
   [(#1162)](https://github.com/PennyLaneAI/catalyst/pull/1162)
   [(#1205)](https://github.com/PennyLaneAI/catalyst/pull/1205)
   [(#1206)](https://github.com/PennyLaneAI/catalyst/pull/1206)
 
-  Using the pipeline, one can run:
+  The `merge_rotations` pass can be provided to the `catalyst.passes.pipeline`
+  decorator:
 
   ```python
-  from catalys.passes import pipeline
+  from catalyst.passes import pipeline
 
   my_passes = {
       "merge_rotations": {}
   }
 
+  dev = qml.device("lightning.qubit", wires=1)
+
   @qjit(circuit_transform_pipeline=my_passes)
-  @qml.qnode(qml.device("lightning.qubit", wires=1))
+  @qml.qnode(dev)
   def g(x: float):
       qml.RX(x, wires=0)
       qml.RX(x, wires=0)
       qml.Hadamard(wires=0)
-      return qml.expval(qml.PauliZ(0))
+      return qml.expval(qml.PauliX(0))
   ```
 
-  Using the Python function, one can run:
+  It can also be applied directly to qjit-compiled QNodes via
+  the `catalyst.passes.merge_rotations` Python decorator:
 
   ```python
   from catalys.passes import merge_rotations
 
   @qjit
   @merge_rotations
-  @qml.qnode(qml.device("lightning.qubit", wires=1))
+  @qml.qnode(dev)
   def g(x: float):
       qml.RX(x, wires=0)
       qml.RX(x, wires=0)
       qml.Hadamard(wires=0)
-      return qml.expval(qml.PauliZ(0))
+      return qml.expval(qml.PauliX(0))
+  ```
+
+* Static arguments of a qjit-compiled function can now be indicated by name via a `static_argnames` argument
+  to the `qjit` decorator.
+  [(#1158)](https://github.com/PennyLaneAI/catalyst/pull/1158)
+
+  Specified static argument names will be treated as compile-time static
+  values, allowing any hashable Python object to be passed to this
+  function argument during compilation.
+
+  ```pycon
+  >>> @qjit(static_argnames="y")
+  ... def f(x, y):
+  ...     print(f"Compiling with y={y}")
+  ...     return x + y
+  >>> f(0.5, 0.3)
+  Compiling with y=0.3
+  ```
+
+  The function will only be re-compiled if the hash value of the static arguments change. Otherwise,
```

Comment:
[Grammar] The subject and verb don't agree here, should be either "the hash value**s** of the static arguments change" or "the hash value of the static arguments change**s**".